### PR TITLE
feat: create reusable resume item

### PR DESCRIPTION
**Pull request by gacetta**
- start process of creating reusable resume item for education & experience sections of resume
- styling / formatting TBD

QUESTION
 - where to store hard coded data for each item?
 - what else will I need?  Awards? Bulletpoints? Images?
 - eventually might need more complex interfaces: `ResumeItem`: `JobItem` | `EducationItem`?

@h15200 do you mind taking a look?  not much to see

### CODE REVIEW

File: src/app/components/ResumeItem.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import { Box, Text, Flex } from "@chakra-ui/react";
+
+interface resumeItem {
```
Comment (h15200) [inline src/app/components/ResumeItem.tsx]: interfaces and types should always be capitalized

File: src/app/components/Resume.tsx [hunk 2]
```diff
@@ -1,24 +1,70 @@
 import { Flex, Text } from "@chakra-ui/react";
 import { SectionHeader } from "./SectionHeader";
-import { ResumeHeader } from "./ResumeHeader";
-import { ResumeSubHeader } from "./ResumeSubHeader";
-import { ResumeDescription } from "./ResumeDescription";
+import { ResumeSection } from "./ResumeSection";
+import { ResumeItem } from "./ResumeItem";
+
+const codesmith = {
```
Comment (h15200) [inline src/app/components/Resume.tsx]: these hardcoded strings should be somewhere else

File: src/app/components/ResumeItem.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  title: string;
+  position?: string;
+  years?: string;
+  description?: string;
+  status?: string;
+}
+
```
Comment (h15200) [inline src/app/components/ResumeItem.tsx:12]: this should be named ResumeItemProps for clarity and in case you need to export them to be used elsewhere

**Comment by h15200** [comment]
In terms of using PRs, you should only write the solution to an existing problem.

If you have additional work, those belong in a ticket like Github Issues. 

So the ticket should have the following:
- existing bug or feature or investigation work that needs to be done
- the progress of the work

The PR should have:
- what the PR does ala "add resume items"
- how tests were added if necessary
- comments on changes to clarify anything for the reviewer
- a link to the corresponding ticket

File: src/app/components/Resume.tsx (hunk 2 above)
Comment (gacetta) [reply src/app/components/Resume.tsx]: created new folder "data" and moved this all into `resumeData.tsx`

**Pull request merged**